.gitlab/testrunner.yml

```diff
@@ -0,0 +1,8 @@
+.testrunner:
+  image: registry.ddbuild.io/images/mirror/dd-trace-py/testrunner:7a2e802af76051f82d698919d2837eff18dbb48e
```

Comment:
don't we want to use the same hash for the test runner image?

Reply:
That's the image _tag_, and that should correspond to the hash used elsewhere to identify the same image 😵‍💫

Reply:
oh got it.